src/emerge.cpp

```diff
@@ -396,7 +396,7 @@ int EmergeManager::getGroundLevelAtPoint(v2s16 p)
 // TODO(hmmmm): Move this to ServerMap
 bool EmergeManager::isBlockUnderground(v3s16 blockpos)
```

Comment:
The parameter `blockpos` can be a const reference. May be offtopic for this pull request.